parts/kubernetesmasteraddons-tiller-deployment1.5.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: v1
```

Comment:
I've been deploying all addons with specific service accounts and associated CRB even if RBAC isn't enabled. That way it all works when RBAC is enabled.